UpdateTemplate.cmd

```diff
@@ -0,0 +1,25 @@
+@ECHO OFF
+PUSHD %~dp0
+
+SETLOCAL
+SETLOCAL ENABLEDELAYEDEXPANSION
+
+:: Check if node exists globally
+WHERE node >NUL
+IF NOT '%ERRORLEVEL%'=='0' (
+    ECHO ERROR: UpdateTemplate.cmd requires node installed globally.
+    GOTO :Exit
+)
+
```

Comment:
No need to check gulp since the step `npm install` will install it and other dependency